src/functionalTest/resources/add-system-admin.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO pi_user (user_id, email, provenance_user_id,user_provenance,roles,forenames,surname)
+VALUES ('87f907d2-eb28-42cc-b6e1-ae2b03f7bba2', 'SyestemAdmin@gmail.com', 'e5f1cc77-6e9a-40ab-8da0-a9666b328464','PI_AAD','SYSTEM_ADMIN','System','Admin');
```

Comment:
Could the email in here also be changed pls

Reply:
Done